windows.md

```diff
@@ -390,6 +390,49 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    If opening Docker Desktop from **Option A or B** shows an error, you may need to enable virtualization on your machine
+
+    The error might look like this:<br><br>
```

Comment:
let's be a more specific here:

- the error message should be written out in text in the first paragraph
- "you may need to" -> "you will need to"
- probably we don't need the 2nd paragraph here

Reply:
https://github.com/upleveled/system-setup/pull/16/commits/643325280ecdf96d3411c6c79c6b08871497fb1e
https://github.com/upleveled/system-setup/pull/16/commits/e0a7242d6d3ef4f70c41f5101d9b7a057481974e
https://github.com/upleveled/system-setup/pull/16/commits/0db74a60c1f8c953e516bf024c0f2c568e7bf199